new/LIT7075QeneHagarena.xml

```diff
@@ -0,0 +1,78 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7075QeneHagarena" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ሐገርነ፡ ኮነት፡ ከመ፡ ሰማርያ፡ እስመ፡ በላእሌሃ፡ ወረደ፡ ነጠብጣብ፡ ዝናም፡ መዓት፤</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Ḥagarǝna konat kama Samāryā ʾǝsma ba-lāʾlehā warada naṭabṭāb zǝnām maʿāt (Qǝne of unknown type)</title>
+                <title xml:lang="en" corresp="#t1">Our city became like Samaria, for over it fell a dripping, rain, wrath…</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="JK"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+            <listWit>
+                <witness corresp="UppQ1524o"/>
+            </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p><foreign xml:lang="gez">Qǝne</foreign> without type indication in the manuscript. It has five lines. 
```

Comment:
With five verses, it should either be Wāzemā or Zayǝʾǝze-type. I do not know, how to distinguish between these two. @DenisNosnitsin1970 surely knows better. If it is possible to identify the type correctly, it can also be noted in the title.